tests/test_wrapers.py

```diff
@@ -0,0 +1,14 @@
+"""Unit tests for the authorization wrappers."""
+
+from unittest.mock import *
+
+
+from fabric8a_auth.errors import AuthError
+import pytest
+
+from fabric8a_auth.auth import login_required
+
+
+def test_user_wrapper():
+    """Test."""
+    pass
```

Comment:
:1st_place_medal:

Reply:
I have removed this file and created missing test cases in test_token_handling.py